### ISSUE TITLE
Did you increase the decoupled weight decay simultaneously when decreasing the learning rate?

### ISSUE THREAD

**Issue by xiangning-chen** [lucidrains/lion-pytorch#2]
Thanks for implementing and testing our lion optimizer!
Just wondering did you also enlarge the decoupled weight decay to maintain the regularization strength?

best,
--xiangning

**Comment by lucidrains** [comment]
@xiangning-chen Hi Xiangning! Thank you for this interesting paper

So far I have been only testing with weight decay turned off. There are a lot of networks that are still trained with just plain Adam, and I wanted to see how Lion fares against Adam alone

**Comment by lucidrains** [comment]
@xiangning-chen but yes, I have noted the section in the paper where you said the weight decay needs to be higher

Let me add that to the readme to increase the chances people train it correctly

**Comment by xiangning-chen** [comment]
Thanks for the update! 
Yeah disabling weight decay for both optimizers is pretty meaning and fair, thank you!

**Comment by lucidrains** [comment]
@xiangning-chen ok good luck! hope this technique holds up to scrutiny!
